### ISSUE TITLE
reprex() not working in RStudio 1.2.1053 desktop windows version

### ISSUE THREAD

**Issue by andresrcs** [rstudio/rstudio#3669]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.2.1053
    OS Version      : Windows 10 Pro
    R Version       : 3.5.1

### Steps to reproduce the problem

- Load the reprex library
- Render a reproducible example with reprex()

### Describe the problem in detail

When I try to render a reproducible example with reprex() I get this error.

![image](https://user-images.githubusercontent.com/14981085/46917638-f7882300-cf8e-11e8-938b-9acf8e632ce9.png)

But with this other setup It's working as expected.

    RStudio Edition : Server
    RStudio Version : 1.2.1053
    OS Version      : Ubuntu Server 16.04.5 LTS
    R Version       : 3.5.1

![image](https://user-images.githubusercontent.com/14981085/46917683-4635bd00-cf8f-11e8-9a38-d775d126fdd6.png)

**Comment by jmcphers** [comment]
A little ironic that a package designed in part to supply reproducible examples of errors is itself downstream of one! :-) This might have to do with the package itself rather than RStudio, but we'll need more details to know for sure.

Are you using the exactly same version of the `reprex` package in both cases?

Can you supply a `traceback()` of the error?

**Comment by andresrcs** [comment]
Yes, I'm using the same version of reprex for both (0.2.1)

![image](https://user-images.githubusercontent.com/14981085/46956446-f8868680-d05a-11e8-9781-19b125831792.png)

This is the traceback
```
10: stop(simpleError(msg, call = sys.call(-1)))
9: stopifnot(no("func") || is.function(func), no("func") || is.list(args), 
       is.character(libpath), is.character(repos), no("stdout") || 
           is.null(stdout) || is_string(stdout), no("stderr") || 
           is.null(stderr) || is_string(stderr), no("error") || 
           is_string(error), is.character(cmdargs), no("echo") || 
           is_flag(echo), no("show") || is_flag(show), no("callback") || 
           is.null(callback) || is.function(callback), no("block_callback") || 
           is.null(block_callback) || is.function(block_callback), 
       no("spinner") || is_flag(spinner), is_flag(system_profile), 
       is_flag(user_profile), is.character(env), no("timeout") || 
           (length(timeout) == 1 && !is.na(timeout)), no("wd") || 
           is_string(wd), no("fail_on_status") || is_flag(fail_on_status))
8: eval(substitute(expr), data, enclos = parent.frame())
7: eval(substitute(expr), data, enclos = parent.frame())
6: with.default(options, stopifnot(no("func") || is.function(func), 
       no("func") || is.list(args), is.character(libpath), is.character(repos), 
       no("stdout") || is.null(stdout) || is_string(stdout), no("stderr") || 
           is.null(stderr) || is_string(stderr), no("error") || 
           is_string(error), is.character(cmdargs), no("echo") || 
           is_flag(echo), no("show") || is_flag(show), no("callback") || 
           is.null(callback) || is.function(callback), no("block_callback") || 
           is.null(block_callback) || is.function(block_callback), 
       no("spinner") || is_flag(spinner), is_flag(system_profile), 
       is_flag(user_profile), is.character(env), no("timeout") || 
           (length(timeout) == 1 && !is.na(timeout)), no("wd") || 
           is_string(wd), no("fail_on_status") || is_flag(fail_on_status)))
5: with(options, stopifnot(no("func") || is.function(func), no("func") || 
       is.list(args), is.character(libpath), is.character(repos), 
       no("stdout") || is.null(stdout) || is_string(stdout), no("stderr") || 
           is.null(stderr) || is_string(stderr), no("error") || 
           is_string(error), is.character(cmdargs), no("echo") || 
           is_flag(echo), no("show") || is_flag(show), no("callback") || 
           is.null(callback) || is.function(callback), no("block_callback") || 
           is.null(block_callback) || is.function(block_callback), 
       no("spinner") || is_flag(spinner), is_flag(system_profile), 
       is_flag(user_profile), is.character(env), no("timeout") || 
           (length(timeout) == 1 && !is.na(timeout)), no("wd") || 
           is_string(wd), no("fail_on_status") || is_flag(fail_on_status)))
4: convert_and_check_my_args(as.list(environment()))
3: callr::r_safe(function(input) {
       options(keep.source = TRUE)
       rmarkdown::render(input, quiet = TRUE, envir = globalenv())
   }, args = list(input = input), spinner = interactive(), stdout = std_out_err, 
       stderr = std_out_err)
2: reprex_render(r_file, std_file)
1: reprex({
       library(ggplot2)
   })
```

**Comment by andresrcs** [comment]
By the way, it was working on build 1.2.1020 

![image](https://user-images.githubusercontent.com/14981085/46958035-84e67880-d05e-11e8-81b8-31e07b7667b8.png)

**Comment by kevinushey** [comment]
This is happening because we're now setting the `repos` option as a list rather than a character vector:

```R
> str(getOption("repos"))
List of 1
 $ CRAN: chr "https://cran.rstudio.com/"
 - attr(*, "RStudio")= logi TRUE
```

A lot of packages expect the `repos` option to explicitly be a character vector so we should find out where this changed.

**Comment by kevinushey** [comment]
I think this issue affects a lot of R codes making this assumption about the repos option:

https://community.rstudio.com/t/pkgdown-build-site-failing-in-version-1-2-1049/16375

@jmcphers, are we planning to publish a new preview soon? I think it's necessary to fix these sorts of problems.

**Comment by jmcphers** [comment]
Yes, once we get a consistent suite of builds (the macOS builder has been out of commission for a few days).

**Comment by kevinushey** [comment]
Great! Since this should be fixed in the latest dailies and hence in the upcoming preview, I'll close this. @andresrcs please feel free to re-open if you're still seeing this issue in the next iteration of the preview release.

**Comment by dfalty** [comment]
Verified fixed on version 1.2.1090.
